xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/uicomponents/viewers/comments.js

```diff
@@ -216,34 +242,43 @@ viewers.Comments = Class.create({
     if (!item) {
       return;
     }
-    item.observe('click', function(event) {
+    item.observe('click', function (event) {
       item.blur();
       event.stop();
+      this.toggleFormHidden();
```

Comment:
Not sure why you need it there? If you clicked on the item then the form is already shown no?

Reply:
@manuelleduc I see that you renamed it but you didn't explain why it's needed there?

Reply:
On some paths, the comments are fully reloaded asynchronously. In this context we need to re-display the form and hide the button.

Reply:
ok thanks.